fixtures/datasources/SFTP/sftp_pass.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: canaries.flanksource.com/v1
+kind: Canary
+metadata:
+  name: sftp-pass
+spec:
+  interval: 30
+  folder:
+    - path: /Users/tarun/flanksource/canary-checker
```

Comment:
you can use `/tmp` for a generic path that exists everywhere

Reply:
Yeah updated... just added for a quick check